src/lib/i18n.tsx

```diff
@@ -83,9 +82,8 @@ export async function dynamicActivate(locale: SupportedLocale) {
     const catalog = await import(`locales/${locale}.js`)
     // Bundlers will either export it as default or as a named export named default.
     i18n.load(locale, catalog.messages || catalog.default.messages)
-  } catch (error) {
-    console.error(error)
-    Sentry.captureException(new Error(`Unable to load locale (${locale})`))
+  } catch (error: unknown) {
+    console.error(new Error(`Unable to load locale (${locale}): ${error}`))
```

Comment:
when should you just `console.error` and when should you do `Sentry.captureException`?

Reply:
In this case, `Sentry.captureException` is duplicating the exception, because it's a symptom of a different issue. Logging adds it to the breadcrumbs in Sentry in case another error is reported.